ooni/errors.py

```diff
@@ -148,6 +149,9 @@ def failureToString(failure):
     elif isinstance(failure.value, ProcessDone):
         string = 'process_done'
 
+    elif isinstance(failure.value, ConnectionDone):
+        string = 'connection_done'
```

Comment:
I'm proposing something like proxy_connection_eof (or equivalent wording). That is slightly more specific than just plain connection_done. And also more accurate since what happened is an unexpected EOF in the connection to the proxy. Makes sense?